### PR TITLE
Fix ocaml and update camlp4 and coq

### PR DESCRIPTION
**Pull request by ncihnegn**
Move ocaml stublibs/*.dll to bin so that camlp4 is usable.
Update camlp4 and coq to the lastest releases 4.02.1+3 and 8.5, respectively.

### CODE REVIEW

**Comment by ncihnegn** [comment]
This fixes Issue #635.

File: mingw-w64-ocaml-camlp4/PKGBUILD [hunk 1]
```diff
@@ -4,39 +4,36 @@ _realname=ocaml-camlp4
...
+patchver=3
+pkgrel=6
 pkgdesc="Pre-Processor-Pretty-Printer for OCaml (mingw-w64)"
 arch=('any')
 url='https://github.com/ocaml/camlp4'
 license=('LGPL2')
 depends=("${MINGW_PACKAGE_PREFIX}-ocaml")
 makedepends=("${MINGW_PACKAGE_PREFIX}-flexdll")
-source=("${_realname}-${pkgver}.tar.gz::https://github.com/ocaml/camlp4/archive/${pkgver}.tar.gz"
+source=("${_realname}-${pkgver}.tar.gz::https://github.com/ocaml/camlp4/archive/${pkgver}+${patch}.tar.gz"
```
Comment (fracting) [inline mingw-w64-ocaml-camlp4/PKGBUILD]: I think you mean `${patchver}` here, not `${patch}`, CI build show a 404 error when downloading from https://github.com/ocaml/camlp4/archive/4.02.1+.tar.gz

**Comment by ncihnegn** [comment]
The appveyor check failed because build time exceeds 1 hour. Anyway to change the limit?

**Comment by mingwandroid** [comment]
We asked ...

**Pull request merged**